src/app.js

```diff
@@ -5,15 +5,10 @@ export default function (app) {
     return ""
   }
 
+  var emit = {event: event, hook: hook}
```

Comment:
If we can find a way to consolidate functions that returns a value (for hooks) and function that don't return a value (event) the yes.

I'll can give it a try, right now I can't think of how to solve this.